### PR TITLE
RELATED: DX-68 Drop v4 and v5 of the GD.UI docs

### PR DESCRIPTION
**Pull request by andriichumak**
There are few reasons to drop old doc versions:
* No customers are using those.
* They are EOS. Likely would not even work with current backend.
* Google still indexes those, creates confusion for users navigating here from search results.
* We want to simplify the repo as much as possible before potential move to Hugo.

Also, added a custom 404 page. Design is "inspired" by 404 on our main website. Currently 404 would show the default GitHub pages site, which is not useful. Adding in same PR, as we are likely to get some traffic to v4 docs from Google search before Google re-index docs.

Doc review is not needed, as there are no changes to the content itself.

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1224/ : SUCCESS in 2m 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/821/ : SUCCESS in 10s

File: website/sidebars.json [hunk 1]
```diff
@@ -4,9 +4,6 @@
             "about_gooddataui",
             "why_gdui",
             "interactive_examples",
-            "whats_new_8",
-            "breaking_changes_8",
-            "migration_guide_8",
```
Comment (andriichumak) [inline website/sidebars.json:9]: Also, I've hidden this 3 pages from the left menu and referenced them directly from `supported_versions` in scope of this PR. We agreed on this change with @no23reason and @kandl on Slack.

File: docs/01_supported_versions.md [hunk 2]
```diff
@@ -23,22 +23,25 @@ Each version of GoodData.UI goes through the following phases:
...
-| :--- | :--- | :--- | :--- | :--- | :--- | :--- |
-| 8 | Generally available | October 8, 2020 | _Not yet defined_ | _Not yet defined_ | _Not yet defined_ | 8.11 |
-| 7 | End-of-Support | May 21, 2019 | October 8, 2020 | March 31, 2022 | _Not yet defined_ | 7.9 |
-| 6 | End-of-Support | November 1, 2018 | May 21, 2019 | October 8, 2020 | _Not yet defined_ | 6.3 |
-| 5 | End-of-Support | April 26, 2018 | November 1, 2018 | May 21, 2019 | _Not yet defined_ | 5.3 |
+| Major Version | Status              | GA               | EOD               | EOS               | Last Minor Version |
+|:--------------|:--------------------|:-----------------|:------------------|:------------------|:-------------------|
+| 8             | Generally available | October 8, 2020  | _Not yet defined_ | _Not yet defined_ | 8.11               |
+| 7             | End-of-Support      | May 21, 2019     | October 8, 2020   | March 31, 2022    | 7.9                |
+| 6             | End-of-Support      | November 1, 2018 | May 21, 2019      | October 8, 2020   | 6.3                |
```
[the remark this answers is not in the document]
Comment (sitina) [reply docs/01_supported_versions.md:36]: Thank you!

File: website/pages/en/404.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                <h1 style={styles.h1}>Ohh, there is nothing in here!</h1>
+              </header>
+              <h2>But you can try</h2>
+              <ul>
+                <li><a href={this.docUrl('about_gooddataui.html')}>About GoodData.UI</a></li>
+                <li><a href={this.docUrl('quickstart.html')}>Start with GoodData.UI</a></li>
+              </ul>
+            </div>
+            <div>
+              <img style={styles.img} src="https://www.gooddata.com/img/pages/error/error-404_animation.svg" alt="Where did it go?" />
```
[the remark this answers is not in the document]
Comment (no23reason) [reply website/pages/en/404.js:40]: @andriichumak good point, I would feel safer copying it, just in case.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1225/ : SUCCESS in 1m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/822/ : SUCCESS in 6s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR577

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1226/ : SUCCESS in 1m 12s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/171/ : SUCCESS in 1m 01s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/366/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul/8/ : FAILURE in 2m 07s (non-voting)

**Pull request merged**